airflow/www/static/js/api/useClearRun.ts

```diff
@@ -17,30 +17,30 @@
  * under the License.
  */
 
-import axios from 'axios';
+import axios, { AxiosResponse } from 'axios';
 import { useMutation, useQueryClient } from 'react-query';
 import { getMetaValue } from '../utils';
 import { useAutoRefresh } from '../context/autorefresh';
 import useErrorToast from '../utils/useErrorToast';
 
-const csrfToken = getMetaValue('csrf_token');
-const clearRunUrl = getMetaValue('dagrun_clear_url');
+const csrfToken = getMetaValue('csrf_token') || '';
+const clearRunUrl = getMetaValue('dagrun_clear_url') || '';
```

Comment:
Should we raise exception when this value is missing? This value seems to be required.

Reply:
You are right, most of the `getMetaValue` are required, defaulting to an empty string is not great. We could update the method to raise instead an exception if not found instead of returning null.

We could parametrize this with something like:
```
function getMetaValue(name: string, required: boolean = true)
```

Reply:
Yeah, setting it to an empty string just kicks the error down the road. I don't know when these are not required, so I don't think we need an extra param. `getMetaValue()` should raise an exception and return empty string by default instead of coding it each time.

Reply:
By the time, `getMetaValue` return type changed to string. This is not necessary anymore.